ui/tests/integration/components/dashboard/overview-test.js

```diff
@@ -111,7 +111,7 @@ module('Integration | Component | dashboard/overview', function (hooks) {
     assert.dom(SELECTORS.cardName('client-count')).doesNotExist();
   });
 
-  test('it should show client count and replication card on enterprise w/ license + namespace enabled', async function (assert) {
```

Comment:
For posterity! Replication card hiding/showing relies on permissions and is tested in the acceptance test added by this PR https://github.com/hashicorp/vault/pull/23452

Reply:
Thank you!